libV2/cache/backend/memory.js

```diff
@@ -0,0 +1,71 @@
+const schema = require('../schema');
+
+class MemoryCache {
+    constructor() {
+        this._data = {};
+        this._shards = {};
+        this._prefix = 'utapi';
+    }
+
+    // eslint-disable-next-line class-methods-use-this
+    async connect() {
+        return true;
+    }
+
+    // eslint-disable-next-line class-methods-use-this
+    async disconnect() {
+        return true;
+    }
+
+    async getKey(key) {
+        return this._data[key];
+    }
+
+    async setKey(key, data) {
+        this._data[key] = data;
+        return true;
```

Comment:
Unless you plan to change the return value to a more meaningful one, you could return the instance itself to be able to chain the calls to set multiple keys or do other stuff: `setKey(...).setKey(...)`

Reply:
This is really just to match the redis client where the return value signifies whether the redis command succeeded